lib/trix/version.rb

```diff
@@ -1,3 +1,3 @@
 module Trix
-  VERSION = '0.9.5'
+  VERSION = '0.9.5'.freeze
```

Comment:
Rubocop was complaining about this, and by default appraisals treat violations as a failure; I'd have to modify that to remove this, up to you.